ooniprobe/Utility/TestLists.mm

```diff
@@ -0,0 +1,72 @@
+#import "TestLists.h"
+#import "NTYCSVTable.h"
+#include <measurement_kit/ooni.hpp>
+#include <measurement_kit/ooni/orchestrate.hpp>
+
+@implementation TestLists
+
++ (id)sharedTestLists {
+    static TestLists *sharedTestLists = nil;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        sharedTestLists = [[self alloc] init];
+    });
+    return sharedTestLists;
```

Comment:
Couldn't there be a race condition between initializing this singleton and using it? @lorenzoPrimi

Reply:
With @lorenzoPrimi we have checked that `dispatch_once` is "absolutely synchronous".